cysync/src/store/hooks/usePortfolio.ts

```diff
@@ -1,4 +1,4 @@
-import { ALLCOINS as COINS, CoinGroup } from '@cypherock/communication';
+import { CoinGroup, COINS } from '@cypherock/communication';
```

Comment:
needs changes in `getAllCoinPriceHistory`

Reply:
Did the changes please check